book/types_of_data.md

```diff
@@ -97,16 +97,18 @@ Dates are in three forms, based on the RFC 3339 standard:
 
 Durations represent a length of time. This chart shows all durations currently supported:
 
-| Duration | Length          |
-| -------- | --------------- |
-| `1ns`    | one nanosecond  |
-| `1us`    | one microsecond |
-| `1ms`    | one millisecond |
-| `1sec`   | one second      |
-| `1min`   | one minute      |
-| `1hr`    | one hour        |
-| `1day`   | one day         |
-| `1wk`    | one week        |
+| Duration  | Length          |
+| --------- | --------------- |
+| `1ns`     | one nanosecond  |
+| `1μs`[^1] | one microsecond |
```

Comment:
While I understand the appeal to follow the general conventions, I think in practical use we should point users to `us` directly as that is what most people can use more easily with international keyboards.

Reply:
```suggestion
| `1μs` or `1us`[^1] | one microsecond |
```
Are you suggesting something like this, where we maintain the footnote, or should I just remove the footnote outright?

Reply:
I'd just keep them both like you have in your suggestion above and remove the footnote.